src/com/homework/MyArrayList.java

```
@@ -0,0 +1,44 @@
+package com.homework;
+
+public class MyArrayList<E> implements List<E> {
+    private int capacity = 10;
+    private int lastElementIndex = -1;
+    private Object[] array;
+
+    public MyArrayList() {
+        array = new Object[capacity];
+    }
+
+    @Override
+    public E get(int index) {
+        return (index >= 0 && index <= lastElementIndex) ? (E) array[index] : null;
```

If index >= 0 && index <= lastElementIndex it better to throw an exception not to return null. As an element in the list could be null. And your user won't be able to understand if the element is null or index is not right.